sdks/python/examples/child/bulk.py

```diff
@@ -0,0 +1,38 @@
+import asyncio
+
+# ❓ Running a Task
+from examples.child.worker import SimpleInput, child_task
+
+child_task.run(SimpleInput(message="Hello, World!"))
+# !!
+
+
+async def main() -> None:
+    # ❓ Bulk Run a Task
+    greetings = ["Hello, World!", "Hello, Moon!", "Hello, Mars!"]
+
+    results = await child_task.aio_run_many(
+        [
+            # run each greeting as a task in parallel
+            child_task.create_bulk_run_item(
+                input=SimpleInput(message=greeting),
+            )
+            for greeting in greetings
+        ]
+    )
+
+    # this will await all results and return a list of results
+    print(results)
+    # !!
+
+    # ❓ Running Multiple Tasks
+    result1 = child_task.aio_run(SimpleInput(message="Hello, World!"))
+    result2 = child_task.aio_run(SimpleInput(message="Hello, Moon!"))
+
+    #  gather the results of the two tasks
+    results = list(await asyncio.gather(result1, result2))
```

Comment:
nit: I don't think we need the `list` here - the code below should work fine with a `tuple`
```suggestion
    results = await asyncio.gather(result1, result2)
```

Reply:
linting was complaining..

Reply:
Fixed [here](https://github.com/hatchet-dev/hatchet/pull/1472/commits/6e73ab58d271876cb423cdc673b6a5babcec7b8f#diff-e52d071bf1d78f66ba72401685a2269263fc331fc7a6022d78c9a0e74a8b1a83R33) - the problem is the mutation of `results` to something of a different type. MyPy doesn't know which type the thing is once you do that (this is dumb but 🤷 )